arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -385,7 +359,7 @@
             "schema": {
               "$ref": "#/definitions/StorageAccountRegenerateKeyParameters"
             },
-            "description": "Specifies name of the key which should be regenerated. key1 or key2 for the default keys"
+            "description": "Specifies name of the key which should be regenerated -- key1 or key2."
```

Comment:
Are `key1` & `key2` names specifically for defaults? If users can have other names then i'd prefer to keep the part of previous statement `for the default keys`. Thoughts?